### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by neha-bhargava**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
+4.21.1
+============
+
+Bug fixes: 
+** Fix the URI for IMDS call to detect region **. This fixes the typo in URI for local IMDS call which is made to detect the region for regional auth.
```
Comment (jennyf19) [inline changelog.txt:5]:
```suggestion
**Fix the URI for IMDS call to detect the region**. This fixes the typo in the URI for local IMDS call which is made to detect the region for regional auth.
```

Comment (jennyf19) [reply changelog.txt:5]: is there an issue you can link to?

**Review by jennyf19 — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (henrik-me) [reply changelog.txt]: Agree would be great with an issue that describes what the issue is, which then links to the PR. It makes things add up meaningfully for those later trying to understand what happened and why.

---
In reply to: [508054053](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2118#discussion_r508054053) [](ancestors = 508054053)

**Review by henrik-me — APPROVED** [review]
:shipit:

**Review by trwalke — APPROVED** [review]

**Comment by henrik-me** [comment]
@neha-bhargava, there is missing links to the actual issue. Can you please ensure that gets added for appropriate tracking?

**Pull request merged**